docs/mindsdb_sql/sql/create/trigger.mdx

```diff
@@ -106,7 +106,7 @@ ON mindsdb_slack.messages
         SELECT 'slack-bot-channel-id' AS channel_id, answer AS text
         FROM chatbot_model m
         JOIN TABLE_DELTA s
-        WHERE  s.user != 'U07J30KPAUF'
+        WHERE  s.user != 'U07J30KPAUF' AND s.channel_id = 'slack-bot-channel-id'
```

Comment:
Today I had to use the LAST keyword in order to answer to only the most recent msgs.

We can add it in like this:
```
WHERE  s.user != 'slack-bot-id'
AND s.channel_id = 'slack-bot-channel-id'
AND s.created_at > LAST
```

Reply:
Done!